lib/cvss_suite/helpers/extensions.rb

```diff
@@ -16,19 +16,14 @@ class Float
   # Since CVSS 3 all float values are rounded up, therefore this method is used instead of the mathematically correct method round().
 
   def round_up(decimal_paces = 0)
-    (self * 10.0**decimal_paces).ceil / 10.0**decimal_paces
+    ceil(decimal_paces).to_f
```

Comment:
@oliverhamboerger : For me this change is legitimate because the ruby "ceil" function has a parameter to handle this correctly.